roles/kubernetes/preinstall/tasks/pre_upgrade.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "Pre-upgrade | check if old credential dir exists"
+  stat: "{{ inventory_dir }}/../credentials"
```

Comment:
Shall we simply point to ``credentials``? This should correctly reference legacy generated folder, see original implementation: https://github.com/kubernetes-incubator/kubespray/commit/57e7a5a34aa416e7c3227f55830291a6e25e0915#diff-7ab4ad936e3bf94fde791e030338479eL99